beacon_node/http_api/tests/tests.rs

```diff
@@ -3271,6 +3271,7 @@ impl ApiTester {
 
         let mut attestation = Attestation {
             aggregation_bits: BitList::with_capacity(duty.committee_length as usize).unwrap(),
+            index: <_>::default(),
```

Comment:
Setting it to the value `0` feels more explicit than using default. use .into() or the type new()